pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -444,11 +449,15 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 		attachableVolumePluginName = attachableVolumePlugin.GetPluginName()
 	}
 
+	// update operation start time cache with fully qualified plugin name for the corresponding volumeToAttach
+	fullPluginName := util.GetFullQualifiedPluginNameForVolume(attachableVolumePluginName, volumeToAttach.VolumeSpec)
+	operationStartTimeCache.UpdatePluginName(createOperationUniqueKey(volumeToAttach.VolumeName, volumeToAttach.NodeName), fullPluginName)
```

Comment:
/assign @davidz627 
to check that the plugin name is correct here in the case of migration.

Reply:
this is correct for migration, since it takes the `VolumeSpec` and the `attachableVolumePluginName` from the above block where we get the correct values for those

Reply:
thanks @davidz627